app/src/test/kotlin/io/intrepid/skotlinton/utils/LiveDataObserverTest.kt

```diff
@@ -0,0 +1,85 @@
+package io.intrepid.skotlinton.utils
+
+import android.view.View
+import android.widget.TextView
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LifecycleRegistry
+import androidx.lifecycle.MutableLiveData
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import io.intrepid.skotlinton.testutils.LiveDataTestBase
+import org.amshove.kluent.shouldEqual
+import org.junit.Before
+import org.junit.Test
+
+class LiveDataObserverTest : LiveDataTestBase() {
+    private val lifecycleOwner = mock<LifecycleOwner>()
+    private val mockObserver = MockObserver(lifecycleOwner)
+    private val lifecycleRegistry = LifecycleRegistry(lifecycleOwner)
+
+    @Before
+    fun setup() {
+        whenever(lifecycleOwner.lifecycle).thenReturn(lifecycleRegistry)
```

Comment:
mockito-kotlin provides a nice dsl for this where we could declare `lifecycleOwner` as
```kotlin
private val lifecycleOwner: LifecycleOwner = mock {
    on { lifecycle } doReturn lifecycleRegistry
}
```
and then lose this line.

Reply:
What do people think about using MockK instead of mockito-kotlin?  It has a lot of extensions and in my opinion has better support for mocking Kotlin code.